# vehicle params
wheelbase: 0.85           # meters
max_velocity: 8.3333      # meters/second
max_steering_angle: 25    # degree
max_brake: 100            # %

# pid controller
AUTO_MODE: 1
USE_PID_CONTROLLER: 1
control_rate: 0.01

vel_pid_mode: 0   # 0: position, 1: delta
vel_kp: 2
vel_ki: 0.01
vel_kd: 1
max_vel_iout: 1

angle_pid_mode: 0
angle_kp: 0.4
angle_ki: 0.05
angle_kd: 1.5
max_angle_iout: 5

brake_threshold: 1  # meters/second
brake_kp: 5

# names of topics to listen to
location_topic: "/gps_pose_vel"
trajectory_topic: "/target_vw"  # "/teb_vel_now"
ctrl_fb_topic: "/ctrl_fb"
io_fb_topic: "/io_fb"

# names of topics to publish to
ctrl_cmd_topic: "/ctrl_cmd"
io_cmd_topic: "/io_cmd"

# names of service
auto_mode_service: "/Begin_stage_control_for_ask"

# data path for teaching mode
teaching_data_path: "./***.csv"

# debug
debug_vel: 0.2
debug_angle: 0